rest/rest-cli/src/test/java/functionaltests/NodeSourceCommandsFunctTest.java

```diff
@@ -99,7 +177,15 @@ public void testCreateNodeSourceAndListNodeSources() throws Exception {
         // source name can appear once per node or more per node, whether the
         // node is deploying or registered (in which case the node source name
         // appears also in the URL of the node)
-        assertThat(nbOccurrencesOfNodeSourceNameInNodeList).isAtLeast(nbNodes);
+        assertThat(nbOccurrencesOfNodeSourceNameInNodeList).isAtLeast(NB_NODES);
+    }
+
+    private void waitForNodeSourceToBeDeployed() {
+        try {
+            Thread.sleep(NODE_SOURCE_DEPLOYED_WAIT_TIME_MILLIS);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this use of "Thread.sleep()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2925)